docs/distributing_cost.md

```diff
@@ -0,0 +1,43 @@
+# Distributing Cost
+Our cost model allows user to distribute the their costs bassed of the cpu or memory usage of their user projects.
+
+## Worker Unallocated Costs:
+
+### How We Calculate
+
+During data processing we add in additional rows to represent the [worker unallocated](https://github.com/project-koku/koku/blob/main/koku/masu/database/trino_sql/reporting_ocpusagelineitem_daily_summary.sql#L503) cost. These rows are added under the project name `Worker unallocated` during the ocp daily summary table creation. Formula:
```

Comment:
```suggestion
During data processing we add in additional rows to represent the [worker unallocated](https://github.com/project-koku/koku/blob/main/koku/masu/database/trino_sql/reporting_ocpusagelineitem_daily_summary.sql#L503) cost. These rows are added under the project name `Worker unallocated` during the OCP daily summary table creation. Formula:
```

Reply:
It might be better to use a permalink here in case this file changes over time and line 503 is different.